index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>To Do List</title>
+        <!-- CSS only -->
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+                integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1"
+                crossorigin="anonymous">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+        <style>
+                .green {
+                        color: lightgreen;
+                }
+
+                .red {
+                        color: lightcoral;
+                }
+
+                .blue {
+                        color: skyblue
+                }
+
+                #itemList {
+                        font-size: 1.5rem;
+                }
+
+                .show {
+                        display: block;
+                }
+
+                .hide {
+                        display: none;
+                }
+        </style>
+</head>
+
+<body>
+        <div class="container">
+                <div class="row">
+                        <div class="col mx-auto col-md-8 mt-3 text-center">
+                                <div class="alert align-items-center" id="message"></div>
+                                <form id="itemForm">
+                                        <h3 class=""> TO DO LIST</h3>
```

Comment:
remove this empty class attribute. And instead of using an empty space use css properties like padding or margin.